lib/ui/view/organization/select_email/select_email_view.dart

```diff
@@ -14,109 +15,93 @@ class SelectEmail extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ViewModelBuilder<SelectEmailViewModel>.nonReactive(
-        viewModelBuilder: () => SelectEmailViewModel(),
-        builder: (context, model, child) => Scaffold(
-              appBar: AppBar(
-                title: Text(
-                  model.getScreenTitle(method),
-                  style: const TextStyle(
-                    fontSize: 16,
-                    color: AppColors.deepBlackColor,
+      viewModelBuilder: () => SelectEmailViewModel(),
+      builder: (context, model, child) => Scaffold(
+        appBar: ZuriAppBar(
+          title: model.getScreenTitle(method),
+          whiteBackground: true,
+          leading: Icons.arrow_back_ios_outlined,
+          leadingPress: () => model.pop()
+        ),
+        body: Container(
+          margin: const EdgeInsets.fromLTRB(8.0, 20.0, 8.0, 0.0),
+          decoration: BoxDecoration(
+              color: Colors.white,
```

Comment:
You have not changed the color yet. 
Use AppColors.whiteColor

Reply:
Done